### PR TITLE
Release 1.6.13

### PR DESCRIPTION
**Pull request by stevemessick**
I'm not sure how the change to `config_dir` ended up here. It was supposed to have been in another PR.

Edit: Removed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Changelog
 ====
+### 1.6.13
+
+* Add --page-ize and --page-token CLI options to all commands that display lists of files.
```
Comment (jplotts) [inline CHANGELOG.md:5]:
```suggestion
* Add --page-size and --page-token CLI options to all commands that display lists of files.

```

File: kaggle/api/kaggle_api_extended.py [hunk 2]
```diff
@@ -290,8 +290,17 @@ class KaggleApi(KaggleApi):
     MAX_NUM_INBOX_FILES_TO_UPLOAD = 1000
     MAX_UPLOAD_RESUME_ATTEMPTS = 10
 
-    config_dir = os.environ.get('KAGGLE_CONFIG_DIR') or os.path.join(
-        expanduser('~'), '.kaggle')
+    config_dir = os.environ.get('KAGGLE_CONFIG_DIR')
```
Comment (jplotts) [inline kaggle/api/kaggle_api_extended.py:293]: Should this be removed, so that this PR just marks the release?

**Review by jplotts — APPROVED** [review]

File: kaggle/api/kaggle_api_extended.py (hunk 2 above)
Comment (stevemessick) [reply kaggle/api/kaggle_api_extended.py:293]: Yes, you're right.  I need to fix another problem, so I'll probably have to do another release anyway.

**Comment by jmasukawa** [comment]
LGTM!

**Pull request merged**